python/packages/ai/teams/ai/augmentations/tools_constants.py

```diff
@@ -0,0 +1,20 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+from __future__ import annotations
+
+# Set of constants used for action tool handling
+
+SUBMIT_TOOL_OUTPUTS_VARIABLE = "temp.submit_tool_outputs"
+# Boolean flag to denote roundtrip tool calls
+
+SUBMIT_TOOL_OUTPUTS_MAP = "temp.submit_tool_map"
+# Map of tool_name to tool_id
+
+SUBMIT_TOOL_OUTPUTS_MESSAGES = "temp.submit_tool_messages"
+# List of tool_output messages
+
+SUBMIT_TOOL_HISTORY = "temp.tool_history"
```

Comment:
are we now storing conversation history in the `conversation` state and in the `temp` state? I think we should find a way to leverage the existing history state instead.

Reply:
we're storing the messages we're sending to/from the model here, but open to leveraging history state instead.

how are we leveraging `state.conversation` right now? I only see it in the `dialogs` logic (`state.conversation[self.id]`) - is this used with the `ConversationHistorySection `class?

Reply:
right now history is persisted using this variable, which your code should use to get all the data it needs. 

https://github.com/microsoft/teams-ai/blob/main/python/packages/ai/teams/ai/planners/action_planner.py#L135
https://github.com/microsoft/teams-ai/blob/main/python/packages/ai/teams/ai/clients/llm_client.py#L37